README.md

```diff
@@ -56,7 +56,7 @@ cd b9 \
 ```sh
 mkdir build \
 && cd build \
-&& cmake -GNinja -DCMAKE_BUILD_TYPE=Debug ../b9 \
```

Comment:
I totally forgot to change it back to in-source build. You will also need to remove the final line of the last step: `cd ..`.

Reply:
OK cool. Do we want people installing esprima from inside `build/`, or from the root src dir? Does it matter?

Reply:
OK, that's what I remembered too. I just have one typo to fix and then you can hit the accept button for me 😁